config/locales/en/models.yml

```diff
@@ -184,6 +189,9 @@ en:
       user_identity:
         one: User identity
         other: User identities
+  task_contribution:
+    model:
+      title_string: Contribution for %{task_title}
```

Comment:
Usually, I am avoiding to include an object type in a method or variable name (doesn't work always, but often). [Here](https://ruby-for-beginners.rubymonstas.org/bonus/good_names.html) are some further examples. Do you think we can update this name to be more self-explanatory?